12-say-grpc/backend/vendor/github.com/campoy/justforfunc/13-cgo-flite/flite/flite.go

```diff
@@ -0,0 +1,55 @@
+package flite
+
+// #cgo CFLAGS: -I /usr/include/flite
+// #include <flite.h>
+// #include <stdlib.h>
+// #cgo LDFLAGS: -L /usr/lib/x86_64-linux-gnu -lflite -lflite_cmu_us_kal -lflite_usenglish
+// extern cst_voice *register_cmu_us_kal(const char *voxdir);
```

Comment:
You don't need to extern this, you don't need to export register_cmu_us_kal. You will never build a library from this package that needs to export this symbol.

Use
```
cst_voice* register_cmu_us_kal(const char*);
```